packages/synapse-react-client/src/components/SynapseTable/SynapseTable.tsx

```diff
@@ -191,11 +194,20 @@ export function SynapseTable(props: SynapseTableProps) {
       // make the rowEntityIDColumnIndex available to all cell renderers
       rowEntityIDColumnIndex,
       rowEntityVersionColumnIndex,
+      renderPlaceholderData: renderTableDataPlaceholder,
+      getWrapInExpandableTd: (cell: Cell<Row, unknown>) => {
+        const selectColumn = selectColumns.find(
+          cm => cm.name === cell.column.id,
+        )
+        return Boolean(
+          selectColumn &&
+            /* JSON handles its own overflow*/
+            selectColumn.columnType !== ColumnTypeEnum.JSON,
+        )
+      },
```

Comment:
Custom rendering logic for placeholder skeletons and wrapping in our expandable `td` component was moved to the shared table UI. Pass these new `meta` args to use them!